wrappers/Python/roadrunner/PyUtils.cpp

```diff
@@ -246,15 +262,16 @@ Variant Variant_from_py(PyObject* py)
 		// Borrowed reference.
 		PyObject* err = PyErr_Occurred();
 		if (err) {
+            char* message = rrGetPyErrMessage();
```

Comment:
the code used to crash here if I specified a value in Python that would overflow in C, so a method was created to print the error message